.circleci/config.yml

```diff
@@ -239,13 +249,17 @@ jobs:
             fi
       - run:
           command: cat release-details/notes.txt
+      - run:
+          name: Tag Docker Image as production release
+          command: |
+            aws ecr get-login-password --region eu-west-2 | docker login --username AWS --password-stdin $ECR_REPO
```

Comment:
Is this already done in a previous step and not necessary?

Reply:
Just the login part, I mean

Reply:
Same workspace but separate stage so is needed it seems - state seems to persist across jobs but not stages

Reply:
And it actually moves to a separate workspace in one of the other PRs so I think it'll need to stay as is